### DISCUSSION TITLE
Having terraform script in same repository as flux configuration?

### DISCUSSION THREAD

**Discussion by dev-samples** [fluxcd/flux2#1459] [Q&A] [answered]
I have based my flux setup on this example:
https://github.com/fluxcd/flux2-kustomize-helm-example

for a bunch of azure aks k8s clusters.

I have my terraform deploy scripts in another repository with a folder for each cluster with its main.tf, variables.tf, etc. files.

I am now considering if I should move my terraform scripts to the flux repository to have everything in one repo under each cluster. I cannot see any reasons not to have it in one repo. Or am I missing something obvious?

**Comment by stefanprodan** [comment, marked as answer]
You can store all your cluster related configs in the same repo, to ignore certain directories that have no meaning to Flux, create a `.sourceignore` file in your repo root and exclude them (docs [here](https://fluxcd.io/docs/components/source/gitrepositories/#excluding-files)).
